kube_proxy/datadog_checks/kube_proxy/kube_proxy.py

```diff
@@ -3,6 +3,29 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from datadog_checks.base.checks.openmetrics import OpenMetricsBaseCheck
 
+METRICS = {
+    'process_cpu_seconds_total': 'cpu.time',
+    'process_resident_memory_bytes': 'mem.resident',
+    'process_virtual_memory_bytes': 'mem.virtual',
+    # Alpha metrics
```

Comment:
all the following metrics are flag as Alpha? (until line24?)
If it is the case, it will be important to add this info in the metrics description in the metadata.csv

Reply:
yes, they are all alpha. i plan to make the metadata.csv updates in a separate PR and indicate the alpha/deprecated metrics similarly to how i did in the kube_apiserver_metrics check .

i'm not seeing a dashboard that uses kubeproxy metrics, do you see one?

Reply:
sounds good!